_sass/base.scss

```diff
@@ -0,0 +1,98 @@
+.site-header {
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	border-top: none;
+	border-bottom-color: transparent;
+	background-color: $dark-background-color;
+
+	min-height: 70px;
+	transition: all .5s;
+
+	display: flex;
+	align-items: center;
+
+	&.site-header-scrolled {
+		min-height: 56px;
+	}
+
+	& > .wrapper {
+		width: 100%;
+		position: relative;
+	
+		.site-title {
+			letter-spacing: unset;
+			color: $text-color-light;
+			text-decoration: none;
+		}
+
+		.site-nav .trigger .page-link {
+			color: $text-color-light;
+			margin-left: 1em;
+			margin-right: 1em;
+			@include media-query($on-palm) {
+				color: $text-color;
+			}
+		}
+
+		@include media-query($on-palm) {
+			margin-left: 0;
+			max-width: calc(100vw - 30px);
+	
+			.site-nav {
+				position: absolute;
+				top: 50%;
+				transform: translateY(-18px);
+				right: $spacing-unit/2;
+			}
+		}
+	}
+}
+
+
+.page-content {
+	margin-top: 4em;
+	
+	.post-content {
+		h1 {
+			font-size: 42px;
+		}
+		hr {
+			margin: 1em auto;
+		}
+	}
+}
+
+.anchorjs-link {
+	transition: all .17s linear;
+	color: #e63;
+	
+	&:hover {
+		text-decoration: none;
+	}
+	
+	&:visited {
+		color: #e63;
+	}
+}
+
+
+.site-footer > .wrapper {
+	display: flex;
+	flex-direction: column;
+
+	.footer-col-wrapper {
+		display: flex;
+		flex-direction: row;
+	}
+}
+
+@import 
```

Comment:
Importing all of these for every page feels like it would result in CSS bloat, but this is probably ignorable for now given that the size of all of the overrides is small

Reply:
Yes, but including styles on a layout-by-layout basis is tricky since all of them inherit from `_layouts/default.html`, which is where all the CSS is included. Before, this was specified with `custom-css`, but that is specified on a page-by-page basis, meaning that in that case, the content wasn't separate from the layout. This is the best compromise I could come up with for this problem, but might have a workaround if we use `{{ page.layout }}` in our includes.